cloud/policies/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Policies in Terramate Cloud
+description: Learn how to detect and prevent misconfigurations of your infrastructure resources with policies in Terramate Cloud.
+---
+
+# Terramate Cloud Policies
+
+Policies in Terramate Cloud help secure your infrastructure by detecting misconfigurations on a per-resource level. They leverage the industry-standard CIS Benchmarks to automatically analyze your infrastructure as code (IaC) changes and deployed resources.
```

Comment:
I would not mentor CIS until defined later on the document.
```suggestion
Policies in Terramate Cloud help secure your infrastructure by detecting misconfigurations on a per-resource level. They leverage the industry-standard benchmarks to automatically analyze your infrastructure as code (IaC) changes and deployed resources.
```

Reply:
or I can link to the following section which introduces CIS?